### PR TITLE
change master to have sel 4 sample

### PR DESCRIPTION
**Pull request by kamal-kaur04**

### CODE REVIEW

File: src/test/resources/conf/single.conf.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "server": "hub-cloud.browserstack.com",
```
Comment (Ankit098) [inline src/test/resources/conf/single.conf.json:2]:
```suggestion
  "server": "hub.browserstack.com",
```

File: src/test/resources/conf/parallel.conf.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "server": "hub-cloud.browserstack.com",
```
Comment (Ankit098) [inline src/test/resources/conf/parallel.conf.json:2]:
```suggestion
  "server": "hub.browserstack.com",
```

File: src/test/resources/conf/parallel.conf.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      "browserName": "firefox"
+    },
+    "safari": {
+      "browserName": "safari",
+      "bstack:options": {
+        "os": "OS X",
+        "osVersion": "High Sierra"
+      }
+    },
+    "ie": {
```
Comment (Ankit098) [inline src/test/resources/conf/parallel.conf.json:27]: Pls confirm is this IE version works with the sample demo site (in some cases demo site breaks on IE)

**Review by Ankit098 — COMMENTED** [review]
Minor suggestions, also if possible let's add setting session status using the js executor..

File: src/test/resources/conf/parallel.conf.json (hunk 3 above)
Comment (kamal-kaur04) [reply src/test/resources/conf/parallel.conf.json]: We were not using IE browser, so removed it..

**Review by Ankit098 — APPROVED** [review]

**Review by francisf — APPROVED** [review]

**Pull request merged**